docs/04.guides/04.cookbooks/24.Query-Handling/page.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Query Handling In Lucee
+id: query-handling-lucee
+---
+
+Here we can find how the lucee supports for SQL queries. 
+
+## Query tags ##
+
+```<cfquery>``` different ways to use the tags in lucee & how we can the passed the value into the query
+
+```lucee
+<cfquery name="qry" datasource="test">
+	select * from Foo1890
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+Above example just retrieve data from database.
+
+
+### Using QueryParam ###
+
+Queryparam used inside the query tag, It used to bind the value with sql statement
+
+```lucee
+<cfquery name="qry" datasource="test">
+	select * from Foo1890
+	where title=<cfqueryparam sqltype="varchar" value="Susi">
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+Passing value with QueryParam, has two advantage,
+
+* Value you passed in queryparam is very secured,
+* Lucee able to cache the statement query statement & reused it as long the value changed.
+
+### params ###
+Here we use param as part of cfquery tag, used to pass the value with SQL statement.
+
+Pass the params value with struct
+
+```lucee
+<cfquery name="qry" datasource="test" params="#{title:'Susi'}#">
+	select * from Foo1890
+	where title=:title
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+referenced as ```:key``` in sql.
+
+Below example to pass the more information with struct
+
+```lucee
+<cfquery name="qry" datasource="test" params="#{title:{type:'varchar',value:'Susi'}}#">
+	select * from Foo1890
+	where title=:title
+</cfquery>
+<cfdump var="#qry#" expand="false">
+```
+
+You can passed the params value by array, It referenced as `?` in SQL
```

Comment:
***toChange***
You can pass the params value using an array. It is referenced as '?' in SQL.